gabbi/tests/gabbits_intercept/backref.yaml

```diff
@@ -73,3 +73,13 @@ tests:
 - name: backref json fail end
   xfail: true
   url: $RESPONSE['url']
+
+- name: get a historical response
+  GET: $HISTORY['post some json'].$RESPONSE['a']
```

Comment:
In py35 `urlunsplit` has slightly different behavior so in order for this test to work properly (and to have a correct url) this needs to be:

    GET: /$HISTORY['post some json'].$RESPONSE['a']